<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SampleReceiveModeDlg</class>
 <widget class="BaseDlg" name="SampleReceiveModeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SampleReceiveModeDlg</string>
  </property>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>样本接收模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>263</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="buttons" stdset="0">
    <string>btn_dlg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>263</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="buttons" stdset="0">
    <string>btn_dlg</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>小时</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="currentText">
    <string notr="true">0.5</string>
   </property>
   <property name="maxVisibleItems">
    <number>8</number>
   </property>
   <item>
    <property name="text">
     <string notr="true">0.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">1.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">2.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">3.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">4.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">5.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">6.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">7.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">8</string>
    </property>
   </item>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>shared/basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>SampleReceiveModeDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
